pkgs/by-name/di/direwolf/package.nix

```diff
@@ -52,33 +54,35 @@ stdenv.mkDerivation rec {
   postPatch =
     ''
       substituteInPlace conf/CMakeLists.txt \
-        --replace /etc/udev/rules.d/ $out/lib/udev/rules.d/
+        --replace-fail /etc/udev/rules.d/ ${placeholder "out"}/lib/udev/rules.d/
       substituteInPlace src/symbols.c \
-        --replace /usr/share/direwolf/symbols-new.txt $out/share/direwolf/symbols-new.txt \
-        --replace /opt/local/share/direwolf/symbols-new.txt $out/share/direwolf/symbols-new.txt
+        --replace-fail /usr/share/direwolf/symbols-new.txt ${placeholder "out"}/share/direwolf/symbols-new.txt \
+        --replace-fail /opt/local/share/direwolf/symbols-new.txt ${placeholder "out"}/share/direwolf/symbols-new.txt
       substituteInPlace src/decode_aprs.c \
-        --replace /usr/share/direwolf/tocalls.txt $out/share/direwolf/tocalls.txt \
-        --replace /opt/local/share/direwolf/tocalls.txt $out/share/direwolf/tocalls.txt
+        --replace-fail /usr/share/direwolf/tocalls.txt ${placeholder "out"}/share/direwolf/tocalls.txt \
+        --replace-fail /opt/local/share/direwolf/tocalls.txt ${placeholder "out"}/share/direwolf/tocalls.txt
       substituteInPlace cmake/cpack/direwolf.desktop.in \
-        --replace 'Terminal=false' 'Terminal=true' \
-        --replace 'Exec=@APPLICATION_DESKTOP_EXEC@' 'Exec=direwolf'
-      substituteInPlace src/dwgpsd.c \
-        --replace 'GPSD_API_MAJOR_VERSION > 11' 'GPSD_API_MAJOR_VERSION > 14'
```

Comment:
This substituteInPlace doesn't match anything anymore. I assume this was to handle an incorrect version check, but I lack the context. Regardless, this should be safe to remove since it wasn't doing anything before anyway.